### DISCUSSION TITLE
Custom last.fm Scrobble Correction

### DISCUSSION THREAD

**Discussion by 15adou44** [ciderapp/Cider#1509] [Feature Request]
For example, album title field: "Title - Single" can be corrected to scrobble as "Title" every time a song is played from it. Or, in song title field: "Zuttomae" can be corrected to "ずっと前". Only other scrobbler I can think of that does this is the web scrobbler, but it's such a useful tool.

**Comment by isthatwyll** [comment]
They should implement the same solution folks on PanoScrobbler app came up with: Letting users set up regex patterns so we can fully customize what is scrobbled from artist name/ track name/ album, etc...
